README.md

```diff
@@ -65,52 +66,58 @@ This step should be done in the case that changes have been made to any of the y
 
 If changes need to be made to the bundle spec, a new bundle can be generated with:
 ```shell script
-operator-sdk generate csv --csv-channel alpha --csv-version $NEW_VERSION --default-channel --operator-name windows-machine-config-operator --update-crds --from-version $PREV_VERSION
+operator-sdk generate csv --csv-version $NEW_VERSION --operator-name windows-machine-config-operator --update-crds --make-manifests=false
```

Comment:
I'm getting a deprecation warning when running this:

```[sebsoto@localhost windows-machine-config-operator]$  operator-sdk generate csv --csv-version 0.0.1 --operator-name windows-machine-config-operator --update-crds --make-manifests=false
INFO[0000] Generating CSV manifest version 0.0.1        
WARN[0005] Required csv fields not filled in file windows-machine-config-operator.v0.0.1.clusterserviceversion.yaml:
	spec.description
	spec.keywords
	spec.provider 
[Deprecation Notice] Package manifests are deprecated. Run `operator-sdk bundle create --generate-only` to create operator metadata. Refer to the version upgrade guide for more information: https://operator-sdk.netlify.com/docs/migration/version-upgrade-guide
```

Reply:
I'm not sure if this command should be ran any more.

Reply:
not sure, ran the command on operator-sdk 0.17
attaching output below. 
```
windows-machine-config-operator pod-permissions ↓2 ↑3 ✅  operator-sdk generate csv --csv-version 0.0.7  --operator-name windows-machine-config-operator --update-crds --make-manifests=false
INFO[0000] Generating CSV manifest version 0.0.7        
INFO[0005] Fill in the following required fields in file windows-machine-config-operator.v0.0.7.clusterserviceversion.yaml:
        spec.description
        spec.keywords
        spec.provider 
INFO[0005] CSV manifest generated successfully          
windows-machine-config-operator pod-permissions ↓2 ↑3 ❔ 1 operator-sdk version
operator-sdk version: "v0.17.0", commit: "2fd7019f856cdb6f6618e2c3c80d15c3c79d1b6c", kubernetes version: "unknown", go version: "go1.13.10 linux/amd64"
```

Reply:
@PratikMahajan what you need to figure out is if running `operator-sdk bundle create` is sufficient.

Reply:
@PratikMahajan please use https://sdk.operatorframework.io/docs/olm-integration/generating-a-csv/ as reference. I do not think `make-manifests` should be set to false